app/controllers/posts_controller.rb

```diff
@@ -3,6 +3,10 @@ class PostsController < ApplicationController
 
   def index
     @posts = Post.recent.published.includes(:tags)
+    respond_to do |format|
+      format.html
+      format.json { render json: @posts }
+    end
```

Comment:
I thought about it, but I don't think the functionality is clear. It looks like it will display the last two blog posts, but maybe it's two random blog posts from the last 10. I opted for doing the smallest change that allows me to implement the feature. I can open an issue to review this later. What do you think?